### DISCUSSION TITLE
Link fixes to issues

### DISCUSSION THREAD

**Discussion by shishkin** [sensedeep/dynamodb-onetable#363] [Ideas]
I've noticed that bug reports often get comments about a bug being fixed but without any link to a commit or PR that contained the fix. It is then difficult to find and understand the code. Maybe it would help not only me if fixes linked back to issues with the [# syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#referencing-issues-and-pull-requests). GitHub is quite good at cross-linking them from both directions. A convention I saw other projects use is to add `Fixes #XYZ` in a PR or commit message to link issues.

**Comment by mobsense** [comment]
Yes, I understand this issue. We've pushed pretty hard on many fronts the last year and competing needs to get releases out quickly and isolate and correlate issues have frequently clashed.

We've been using the # syntax to refer to issues and PRs more recently. But not cross reference commits.  The issue is manpower. If we could get some more contributors in this regard, it would really help.

Want to help?

**Reply by shishkin** [reply]
Where exactly do you need help? I assumed this is a rather question of discipline, project governance and contributor guidelines. Not sure I have time to commit to police contributions. But I can help brainstorm the concept with you.

**Reply by mobsense** [reply]
Just help in general. All contributions of effort are greatly received.

**Reply by shishkin** [reply]
Sure, I can help where I can.
